singlestoredb_cloud/metadata.csv

```diff
@@ -0,0 +1,22 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,integration,description,orientation,short_name,curated_metric
+singlestoredb_cloud.cpu_usage_seconds_total,gauge,60,,,singlestoredb_cloud,Cumulative cpu time consumed in core-seconds,0,,
+singlestoredb_cloud.cpu_resource_limits,count,60,,,singlestoredb_cloud,The number of requested CPU resource limit,0,,
+singlestoredb_cloud.mem_used_b,count,60,byte,,singlestoredb_cloud,The memory used by all processes of SingleStoreDB Cloud,0,,
+singlestoredb_cloud.mem_total_b,count,60,byte,,singlestoredb_cloud,The memory limit for the SingleStoreDB Cloud node_id based on its cgroup membership in bytes.,0,,
+singlestoredb_cloud.disk_used_b,count,60,byte,,singlestoredb_cloud,The total bytes used by the disk,0,,
+singlestoredb_cloud.disk_total_b,count,60,byte,,singlestoredb_cloud,"The total size of the disk, in bytes",0,,
+singlestoredb_cloud.failed_write_queries,gauge,60,query,,singlestoredb_cloud,Number of write queries which have failed,0,,
+singlestoredb_cloud.successful_write_queries,gauge,60,query,,singlestoredb_cloud,Number of write queries which have successfully run,0,,
+singlestoredb_cloud.failed_read_queries,gauge,60,query,,singlestoredb_cloud,Number of read queries which have failed,0,,
+singlestoredb_cloud.successful_read_queries,gauge,60,query,,singlestoredb_cloud,Number of read queries which have successfully run,0,,
+singlestoredb_cloud.rows_affected_by_writes,gauge,60,row,,singlestoredb_cloud,Number of rows affected by write queries,0,,
+singlestoredb_cloud.rows_returned_by_reads,gauge,60,row,,singlestoredb_cloud,Number of rows returned by read queries,0,,
+singlestoredb_cloud.Execution_time_of_write,gauge,60,millisecond,,singlestoredb_cloud,Total wall-time of all writer queries since server startup.,0,,
+singlestoredb_cloud.Execution_time_of_reads,gauge,60,millisecond,,singlestoredb_cloud,Total wall-time of all read queries since server startup.,0,,
+singlestoredb_cloud.Threads_connected,count,60,thread,,singlestoredb_cloud,The number of connections currently open.,0,,
+singlestoredb_cloud.max_connections,count,60,thread,,singlestoredb_cloud,The number of simultaneous clients allowed,0,,
+singlestoredb_cloud.Threads_running,count,60,thread,,singlestoredb_cloud,The number of threads that are currently active.,0,,
+singlestoredb_cloud.max_connection_threads,count,60,thread,,singlestoredb_cloud,The maximum number of kernel threads for processing queries.,0,,
+singlestoredb_cloud.ready_queue,count,60,query,,singlestoredb_cloud,The number of connections with pending queries not associated with threads.,0,,
+singlestoredb_cloud.aborted_connects,gauge,60,connection,,singlestoredb_cloud,The number of failed attempts to connect to the server.,0,,
+singlestoredb_cloud.connections,gauge,60,connection,,singlestoredb_cloud,The number of successful or non-successful connection attempts to the server.,0,,
```

Comment:
```suggestion
singlestoredb_cloud.connections,gauge,60,connection,,singlestoredb_cloud,Number of successful and failed connection attempts to the server,0,,
```

Reply:
I'm not sure if this suggestion is technically accurate, but it looks like this metric does not distinguish between successful and failed connections -- so I'm guessing that it's cumulative of both.
Please let me know if that's not correct!